packages/website/src/pages/styles.module.css

```diff
@@ -33,6 +33,20 @@
   margin: 1rem 1rem;
 }
 
+.button_primary {
```

Comment:
CSS modules use camelCase by convention

Reply:
Sure! I have just pushed a new commit to fix that. 
Thanks for reviewing it!